### ISSUE TITLE
[Docs] Replace Github Emojis with their image equivalents

### ISSUE THREAD

**Issue by AVGP** [GoogleChrome/rendertron#321]
In `docs/index.md` there are three emojis (for example `:hammer:`). They are shown correctly in the preview (for example like this: :hammer: ) but the HTML rendering at https://googlechrome.github.io/rendertron doesn't show them correctly.

1. Look at the preview in https://github.com/GoogleChrome/rendertron/blob/master/docs/index.md and right click an emoji to get the image address
2. Edit `docs/index.md` to use the image instead of the `:hammer:` etc.

To do (2) you will need to replace the emojis (for example `:hammer:`) with the corresponding image (for example `![](https://github.githubassets.com/images/icons/emoji/unicode/1f528.png)`)

**Comment by mfaheemakhtar** [comment]
Hey @AVGP, The image is bigger than the emoji. 

:hammer: `:hammer:`

![](https://github.githubassets.com/images/icons/emoji/unicode/1f528.png)
`![](https://github.githubassets.com/images/icons/emoji/unicode/1f528.png)`

<img src="https://github.githubassets.com/images/icons/emoji/unicode/1f528.png" width="16"> `<img src="https://github.githubassets.com/images/icons/emoji/unicode/1f528.png" width="16">`

**Comment by yogesh-io** [comment]
hey @AVGP 
I would like to work on this issue.

**Comment by yogesh-io** [comment]
hey @AVGP,
create a PR for this, please have a look.

**Comment by AVGP** [comment]
@mfaheemakhtar That's correct! Good catch! 👍 

I just merged the PR that replaces them with markdown for images - would you like to do another PR to use the HTML markup you mention?

**Comment by mfaheemakhtar** [comment]
@AVGP , sure.

**Comment by mfaheemakhtar** [comment]
@AVGP,  there are a couple of issues here.

1. <img class="emoji" src="https://github.githubassets.com/images/icons/emoji/unicode/1f528.png" width="20" height="20"> The `img` tag is being wrapped in an `a` tag. One way is to set `href` to `#`.

2. When using the actual emoji, it is being replaced by an `g-emoji` element, which has its own styles plus it sets the height and width to '1rem'.

TLDR: The position and size will not be absolutely same as the emoji. 

I'm making the PR, you can compare if it looks good or not and merge.
